templates/nextjs/src/components/WalletDisplay.tsx

```diff
@@ -0,0 +1,40 @@
+import toast from "react-hot-toast";
+import type { WalletUnlocked, WalletLocked, BN } from "fuels";
+
+const getTruncatedAddress = (address: string) => {
+  return address.slice(0, 6) + "..." + address.slice(-4);
+};
+
+const copyToClipboard = (text: string) => {
+  navigator.clipboard.writeText(text);
+  toast.success("Address copied to clipboard");
+};
+
+export const WalletDisplay = ({
+  wallet,
+  walletBalance,
+}: {
+  wallet?: WalletUnlocked | WalletLocked;
+  walletBalance?: BN;
+}) => {
+  if (!wallet) {
+    return null;
+  }
+
+  return (
```

Comment:
You can do return `null && <Component/>` here.

Reply:
Lol, that won't work haha, there was some other pattern around `null` but I forgot